ibm/service/secretsmanager/data_source_ibm_sm_public_certificate_metadata.go

```diff
@@ -253,16 +253,6 @@ func DataSourceIbmSmPublicCertificateMetadata() *schema.Resource {
 							Computed:    true,
 							Description: "Determines whether Secrets Manager rotates your secret automatically.Default is `false`. If `auto_rotate` is set to `true` the service rotates your secret based on the defined interval.",
 						},
-						"interval": &schema.Schema{
```

Comment:
Kindly, update documentation accordingly. Remove respective attribute references from docs.

Applies to all the resources/datasources where ever you have removed a field from schema.

Reply:
Fixed the documentaion